api/buildout.cfg

```diff
@@ -29,6 +29,7 @@ eggs =
     robotframework-debuglibrary
     robotframework-react
     robotframework-seleniumlibrary
+    robotframework-selenium2library
```

Comment:
@datakurre I am happy to make a robotframework-react release that depends on robotframework-seleniumlibrary.

Reply:
@tisto At the end the issue was not a missing dependency, but ReactLibrary expected to find imported instance of "Selenium2Library". Probably it should look for an instance of SeleniumLibrary at first and then look for Selenium2Library (since it takes some time that all Selenium2Library users have moved back to SeleniumLibrary).

Reply:
@tisto Also, I did not get ReactLibary working yet. At first, it didn't do anything (I was missing reducer-argument). When I called it with reducer=userSession, it failed with ``JavascriptException: Message: TypeError: window.appStore is undefined``.

Reply:
@datakurre done: https://github.com/kitconcept/robotframework-react/pull/9